web/client/src/library/components/graph/help.ts

```diff
@@ -664,10 +665,9 @@ function getUpdatedNodes(
   connectedNodes: Set<string>,
   selectedNodes: Set<string>,
   connections: Map<string, Connections>,
-  withConnected: boolean = true,
-  withImpacted: boolean = true,
-  withSecondary: boolean = true,
-  withColumns: boolean = true,
```

Comment:
`withColumns` triggers rebuilding and repositioning of the lineage sinse node with columns has different heigh, so no need for additionally update